### DISCUSSION TITLE
Removing data from line chart

### DISCUSSION THREAD

**Discussion by powertec-dan** [vikramlearning/blazorbootstrap#388] [Q&A] [answered]
Hello,

I'm trying to build some charts that have scrolling data over a particular timespan. As new data comes in, the oldest data should get removed.

I went looking for a method to remove data (and labels) but didn't see one.
I can completely update the chart by replacing the data, but I was hoping for something a little less heavy.

What is the correct way to remove data from a chart?

Cheers,
Dan

**Comment by gvreddy04** [comment, marked as answer]
@powertec-dan We do not currently have a **remove** feature. Please create an issue and we will add it. For now, please use the **update** method.

**Reply by powertec-dan** [reply]
Thanks for the quick reply. I've added: https://github.com/vikramlearning/blazorbootstrap/issues/391
